call-notes/call_6.md

```diff
@@ -0,0 +1,160 @@
+
+So, I think we are at 20 minutes now. I don't know whether I have introduced the new method enough so everyone has good grabs. But, we could we could see if anyone has any questions at this point. Hello, or I can proceed if no one has questions.
+
+**Jacob Creech:** Does anybody have any questions currently on the current approach or should we just continue?
+
+Alright go ahead and continue then `Wen`.
+
+#### Exit Silent Repair Phase
+
+Video |  [18:26](https://youtu.be/GA5AVg_svj8?t=1106)
+-|-
+
+**Wen Xu:** Okay! So, to exit a silent repair phase. I think one of the most important thing in outage handling is to make sure everyone is on the same page. Otherwise, you think everyone is on the same page, you single-handedly enter the restart and start making new blocks while everyone still repairing blocks, that would be disastrous. And a lot another I audit might be happening. So here we are very careful when we exit the silent repair phase. We would count whether enough people are ready for action. So the current check is whether 80% of the validators receipt the heaviest fork response from 80 percent of people. So we count 80% here because this is the current line we draw when we do a restart. We wait for 80% to join the restart, then we proceed. And also, even if we see that 80% of people respond, got response from 80%, we also linger for maybe we haven't totally decide the numbers here, we would say linger for two minutes because a gossip message propagation takes time. So linger for two minutes so that my HeaviestFork which contains how many responses I got can reach everyone.
+
+And then, you can perform security checks. Check one is, did everyone agree on the same block, which means same slot and same hash. And whether I also have my local optimistically confirmed block before the restart right. So because my local confirmed block means before the restart, I saw two-thirds of the votes on this block already. So you would imagine this block should be on the selected Fork. So it should be the ancestor or it should be the selected block. If it's not the case then the security check fails because something is wrong, my local confirmed block is getting roll back then you would also exit and hold and wait for human inspection. And if every check succeeds, then perform the current restart logic. We probably will clear the gossip CRDS table so that we don't carry the old restart messages into the new environment. And then, we would automatically start snapshot creation. In contrast to what we are doing today is we manually ask people to manually do it using Ledger. Here we might start earlier once we find out everyone agrees on the same block. Then we execute the same logic, what we are doing now in this new arguments.
```

Comment:
```suggestion
And then, you can perform security checks. Check one is, did everyone agree on the same block, which means same slot and same hash. And whether I also have my local optimistically confirmed block before the restart right. So because my local confirmed block means before the restart, I saw two-thirds of the votes on this block already. So you would imagine this block should be on the selected Fork. So it should be the ancestor or it should be the selected block. If it's not the case then the security check fails because something is wrong, my local confirmed block is getting roll back then you would also exit and halt and wait for human inspection. And if every check succeeds, then perform the current restart logic. We probably will clear the gossip CRDS table so that we don't carry the old restart messages into the new environment. And then, we would automatically start snapshot creation. In contrast to what we are doing today is we manually ask people to manually do it using Ledger. Here we might start earlier once we find out everyone agrees on the same block. Then we execute the same logic, what we are doing now in this new arguments.
```